### PR TITLE
add umbrel-5 migration action

### PR DESCRIPTION
**Pull request by islandbitcoin**

### CODE REVIEW

**Comment by chrisguida** [comment]
Seems like it should work, let me know when successfully tested.

**Review by elvece — APPROVED** [review]
Can't speak to functionality, but looks ok to me

**Comment by islandbitcoin** [comment]
> 

thanks! waiting test before merging to master

**Comment by islandbitcoin** [comment]
test failed... looks like eos v0.3.3 changed something in containers that broke both this action and the 0.4 migration action.

**Comment by islandbitcoin** [comment]
both 0.4 and 0.5 migration actions are tested and working now.

File: manifest.yaml [hunk 1]
```diff
@@ -191,12 +191,11 @@ actions:
     input-spec:
       umbrel-host:
         type: string
-        name: Umbrel Hostname
-        description: "The hostname you use to log into your Umbrel dashboard or SSH"
+        name: Umbrel IP Address
+        description: "The IP Address for your Umbrel. You can find this by running the command `ping umbrel.local` while connected to your LAN."
         masked: false
-        placeholder: "umbrel.local"
+        placeholder: "192.168.1.9"
```
Comment (chrisguida) [inline manifest.yaml:197]: Kind of an odd placeholder. But I guess it does the job

**Pull request merged**